README.md

```diff
@@ -93,12 +93,50 @@ On the [Projects Page](https://github.com/Free-Code-Camp-Indy/free-code-camp-ind
 
   * Save file with a `.sass` extension and precede filename with an `_` (eg `_filename.sass`)
   * Import file into `style.sass` file (eg `@import "filename"` - `_` and extension not required)
-  
+
 1. CSS or SCSS syntax files:
 
   * Save file with a `.scss` extension and precede filename with an `_` (eg `_filename.scss`)
   * Import file into `style.sass` file (eg `@import "filename"` - `_` and extension not required)
-  
+
+## Generate new mock "events" data
+By hosting through git pages we've lost the ability to use a server-side to hide away secret API keys.  You should not commit secret API keys to a public repository. While we figure out a trusted solution, we must generate data and upload a ".mock.json" file locally using your own API key.  Note: you only have a limited number of API requests per day so don't send too many requests.
+
+Here's how you set up to generate these files.
+
+#### The Easy Way
+1. Acquire an API key from Meetup.com by [visiting meetup's api key page](https://secure.meetup.com/meetup_api/key/)
+2. Add this key to `meetup_api_key.txt`, a file in the root of this repository
+3. Run `npm run generate`
+
```

Comment:
This is great, having the easy and the hard way is good and educational. The only thing I'd request changed is to follow the markdown numbering convention we are using, where each number list item starts with `1.`.

```
1. item
1. item
1. item
```
instead of
```
1. item
2. item
3. item
```
It just makes Git diffs easier and they both render the same.